packages/ui/src/components/Datepicker/helpers.ts

```diff
@@ -110,7 +112,8 @@
     defaultOptions = options;
   }
 
-  return new Intl.DateTimeFormat(language, defaultOptions).format(date);
+    return DateFNSFormat(new Intl.DateTimeFormat(language, defaultOptions).format(date), dateFormat);
+  
 };
 
 export const startOfYearPeriod = (date: Date, years: number): number => {
```

Comment:
👆 I think that this comment from the AI makes sense.

Reply:
> 👆 I think that this comment from the AI makes sense.


I have made the way AI is suggesting @rluders , but I want your suggestion is that, `date-fns` package have the list/supported `locale`, how can we convey it to the user that this can be the correct locale?

Reply:
@rluders I've added the note in the documentation regarding the `date-fns`.

Furthermore, I've made an extra support/change is that in the `DatepickerProps` is that I've added the list of available locales from the `date-fns` so when the user wants to use `language=''` in the `Datepicker` they will get the list of available locales to take from.

Reply:
@rluders your input please on this